tests/conftest.py

```diff
@@ -1,26 +1,13 @@
 import hashlib
-import pathlib
 import platform
-import shutil
 import ssl
-import tempfile
 
 import pytest
 import trustme
 
 pytest_plugins = ['aiohttp.pytest_plugin', 'pytester']
 
 
-@pytest.fixture
-def shorttmpdir():
```

Comment:
IIRC thee fixture exists to satisfy cases when `path is too long` was reported by CI.
If travis/appveyor are happy without the fixture -- that's fine to me.
We can reinvent it later if needed.

Reply:
If this needs some fix later again it should be considered to get fixed in pytest also.
(Some quick thought: maybe the (temp) base dir could be shortened on Travis if that causes trouble, e.g. by exporting TMPDIR or something alike.)